config/components/manager/manager.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: manager
+  namespace: system
+  labels:
+    control-plane: controller-manager
```

Comment:
That's seems to be too general labels which may overlap with other generated controllers. What if we use here something devworkspace specific?

Reply:
I mostly left it because all kubebuilder projects are generated with this label and I didn't see anything about it in the documentation. We can probably remove it.

Reply:
I've removed the labels now and we're using only the ones from before. I still don't know what the purpose of the label is supposed to be, though.